pages/common/mdn.md

```diff
@@ -0,0 +1,15 @@
+# mdn
+
+> Quick tool for search in Mozilla Developer Network from command line
+
+- List a few results:
+
+`mdn concat`
+
+- List a few results with topic=js:
+
+`mdn -t js string`
```

Comment:
Is there a long form of this argument at all? Also, I think there's some missing token syntax here (it's the `{{` and `}}` - clients use this to detect when something's editable by the user)
```suggestion
`mdn -t {{js}} {{string}}`
```